features/javascript/response.feature

```diff
@@ -0,0 +1,76 @@
+@javascript
+Feature: Response validation
+
+  Background:
+    Given you call:
+    """
+    const gavel = require('gavel');
+    """
+    And you define following HTTP response object:
+    """
+    const real = {
+      "statusCode": "200",
+      "statusMessage": "OK",
+      "headers": {
+        "content-type": "application/json",
+        "date": "Wed, 03 Jul 2013 13:30:53 GMT",
+      "server": "gunicorn/0.17.4",
```

Comment:
This and the below headers are mis-indented.

```suggestion
        "server": "gunicorn/0.17.4",
```

Reply:
Fixed in 8796e68.